examples/hangman.rs

```diff
@@ -0,0 +1,274 @@
+//! A hangman game to guess the hidden word.
+// To run this example, use the command: cargo run --features unstable --example hangman
+#[cfg(all(not(feature = "unstable")))]
+compile_error!("This example relies on unstable features. Run with `--features unstable`");
+
+use std::io::{Error, Write};
+
+use graphic::HangmanDesign;
+use turtle::{rand::RandomSlice, Drawing, Point, Speed, Turtle};
```

Comment:
Instead of importing the `RandomSlice` trait, you could import the `choose` function directly and use that. The `turtle::rand` module is currently a little different than the version on docs.rs/turtle. I suggest running `cargo doc --open` to see the current docs.

Reply:
Or import nothing and call by fully qualified path?

Reply:
Either works but given that `choose` doesn't clash with anything else in scope, I think it's fine to just import it. You could also import `rand` and use `rand::choose(...)`. `turtle::rand::choose(...)` is a bit long.

Reply:
done!